src/SFA.DAS.ApprenticeCommitments.Web/Services/OuterApi/Apprenticeship.cs

```diff
@@ -33,10 +33,10 @@ public class Apprenticeship
     public enum DeliveryModel
     {
         [Display(Name = "Normal")]
-        Normal = 0,
+        Regular = 0,
```

Comment:
Is the display name going to show Normal?

Reply:
Thanks, you're right it should be "Regular."  Done.